### ISSUE TITLE
Cannot compile with Android SDK version >= 31

### ISSUE THREAD

**Issue by SerggioC** [paypal/android-checkout-sdk#76]
**Describe the bug**
App does not compile with android SDK >= 31.

**To Reproduce**
1. In app build.gradle file set compileSdkVersion 31 and targetSdkVersion 31. 
2. compile.

**Expected behavior**
App should compile.

**Error**
Manifest merger failed with multiple errors, see logs.
Error:
	android:exported needs to be explicitly specified for <activity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined.

**Smartphone (please complete the following information):**
 - Paypal SDK Version 'com.paypal.checkout:android-sdk:0.5.2'

**Comment by mauro-poq** [comment]
Hey @SerggioC I was looking this just today, there will be a fix in the next version as described (here)[https://github.com/paypal/android-checkout-sdk/issues/67]

**Comment by SerggioC** [comment]
> Hey @SerggioC I was looking at this just today, there will be a fix in the next version as described in #67

Hey, alright lets wait for the fix.

**Comment by GengJian** [comment]
Please merge this pull-request:   https://github.com/paypal/android-checkout-sdk/pull/77

**Comment by andrespelaezp** [comment]
Hi @SerggioC, thanks for reporting and working on a solution for this issue. 

I will check with the team if the solution is already on place on our side, and if not, I will take your solution for a test and include it for the next release. 

Thanks again!

**Comment by SerggioC** [comment]
Hi @andrespelaezp,

Pull request is from @GengJian.
Anyway when is your next release?

Thank you.

**Comment by andrespelaezp** [comment]
@SerggioC @GengJian thank you both.

Our main focus for the next release is support for Android 12. As of now, we don't have an estimated time but we are working to have it ready ASAP.

**Comment by Onheil13** [comment]
Hi, @andrespelaezp.

I would like to ask what is the estimated date/time of the next release?

Thank you.

**Comment by tdchow** [comment]
Hey @Onheil13, @SerggioC, @GengJian and @mauro-poq!

Good news! We just released version 0.6.0 of the Checkout SDK which now targets Android 12. 

```implementation 'com.paypal.checkout:android-sdk:0.6.0'```

@SerggioC - once you've verified the new version, would you be able to close out this issue? Thanks!

**Comment by tdchow** [comment]
@SerggioC - no problem! I saw that you created a new issue for the crashes you're seeing. Let's use that issue to unblock your integration. Closing this one out.

**Comment by Onheil13** [comment]
Thanks for the update.
